frontend/src/Editor/CommentNotifications/index.jsx

```diff
@@ -11,28 +11,39 @@ import TabContent from './Content';
 import useRouter from '@/_hooks/use-router';
 
 const CommentNotifications = ({ socket, toggleComments, appVersionsId }) => {
-  const [notifications, setNotifications] = React.useState([]);
+  const [unresolvedNotifications, setUnresolvedNotifications] = React.useState([]);
+  const [resolvedNotifications, setResolvedNotifications] = React.useState([]);
   const [loading, setLoading] = React.useState(false);
   const [key, setKey] = React.useState('active');
 
   const router = useRouter();
 
-  async function fetchData(k) {
-    const isResolved = k === 'resolved';
```

Comment:
@mslourens I believe the issue exists when the user switches to the "resolved" tab, feel like the issue can be solved by adding another check for `isResolved` based on the tab `key` value and prevent making an extra api call i.e

```js
const isResolved = k === 'resolved' || key === 'resolved';
```